scp/roles.php

```diff
@@ -39,11 +39,11 @@
             $msg = __('Role updated successfully');
         } elseif ($errors) {
             $errors['err'] = $errors['err'] ?:
-                sprintf(__('Unable to update %s. Correct error(s) below and try again!'),
+                sprintf(__('Unable to update %s. Correct any errors below and try again.'),
```

Comment:
Sometimes its "Correct ANY errors..." and sometimes its "Correct THE errors..." (e.g. /scp/tasks @ Line103). I guess it'd make sense to just use one phrase / sentence and not several to hint at errors which shall be corrected ;)
Rest of this PR looks pretty good so far :+1: